Src/Codge.Generator/Presentations/ModelLoader.cs

```diff
@@ -13,19 +15,20 @@ public ModelLoader(ILogger logger)
             _logger = logger;
         }
 
-        public ModelDescriptor LoadModel(string path, string modelName)
+        public ModelDescriptor LoadModel(IReadOnlyCollection<string> paths, string modelName)
         {
-            _logger.LogInformation("Loading model [{path}]", path);
+            _logger.LogInformation("Loading model [{0}]", string.Join(", ", paths));
```

Comment:
Can you please try providing 'paths' without Join? I believe Serilog destructures collections.

Reply:
you are right: output like:
Loading model [["C:\\Dev\\Codge\\tests\\Codge.Generator.Test\\TestStore\\XsdModel\\Model.xsd", "b.xsd"]]